Makefile

```diff
@@ -1805,8 +1805,15 @@ fuzz: bin/fuzz
 	bin/fuzz $(TESTFLAGS) -tests $(TESTS) -timeout $(TESTTIMEOUT) $(PKG)
 
 # Short hand to re-generate all bazel BUILD files.
+#
+# Even with --symlink_prefix, some sub-command somewhere hardcodes the
+# creation of a "bazel-out" symlink.  This bazel-out symlink can only
+# be blocked by the existence of a file before the bazel command is
+# invoked. This is that the rule here does.
+#
 bazel-generate: ## Generate all bazel BUILD files.
 	@echo 'Generating DEPS.bzl and BUILD files using gazelle'
+	rm -f bazel-out && echo "placeholder file - auto-generated" >bazel-out
```

Comment:
I don't think we need to automate this -- a user who does not want `bezel-out` can simply do this by hand _once_ and bazel won't overwrite the regular file?

Reply:
^